packages/generator/src/renderer.ts

```diff
@@ -155,15 +135,19 @@ export function loadApiConfig(
 export async function processApiFamily(
   apiFamily: RestApi[],
   inputDir: string
-): Promise<model.document.Document[]> {
+  // TODO: Once logic is moved to raml-toolkit this should just be an optional field on RestApi
```

Comment:
Please add this to that story and add a story number here for tracking

Reply:
Actually no,  going to remove this line :-p This was actually for a new parameter for renderer that I ended up deleting as it wasn't needed yet for my story.  So Ill just remove this comment.

Reply:
This function will go away in the story to use the raml-toolkit refactor anyway. ApiGroup.read() pretty much handles all of this. This looks good right now.

Reply:
That's what I figured. Which is why the comment specifically says "move the logic" I don't care about the function at all.